### ISSUE TITLE
getPath('documents') returns wrong path after bundling

### ISSUE THREAD

**Issue by designdoughdev** [electron/electron#16652]
* Output of `node_modules/.bin/electron --version`: 4.0.1
* Operating System (Platform and Version): Windows 10 1803

**Expected Behavior**
`remote.app.getPath('documents')` to return the users Documents directory

**Actual behavior**
In development returns `C:\Users\designdough\Documents\image.jpg`, after building with Electron-Forge and Squirrel Installer it returns `file:///C:/Users/designdough/AppData/MyApp/app-0.1.1/resources/app/src/C:\Users\designdough\Documents\image.jpg` in the tooltip, but in the `src="C:\Users\designdough\Documents\image.jpg"` and the image is missing.

**To Reproduce**
It's a private app.

**Screenshots**

**Additional Information**

**Comment by sofianguy** [comment]
> * Output of `node_modules/.bin/electron --version`: 6.4.1

Hi @designdoughdev. `6.4.1` is not one of Electron's versions. 
What Electron version did you use and what is the last known working Electron version?

**Comment by designdoughdev** [comment]
Hi @sofianguy, I've updated my original question now.

**Comment by designdoughdev** [comment]
@sofianguy Any updates on this? It's the final thing now before we can release the app to our client.

Thanks

**Comment by electron-triage** [comment]
Thank you for taking the time to report this issue and helping to make Electron better.

The version of Electron you reported this on has been superseded by newer releases. 

If you're still experiencing this issue in Electron 6.x.y or later, please add a comment specifying the version you're testing with and any other new information that a maintainer trying to reproduce the issue should know.

I'm setting the `blocked/need-info` label for the above reasons. This issue will be closed 7 days from now if there is no response.

Thanks in advance! Your help is appreciated.

**Comment by electron-triage** [comment]
Thank you for your issue!

We haven't gotten a response to our questions in our comment above. With only the information that is currently in the issue, we don't have enough information to take action. I'm going to close this but don't hesitate to reach out if you have or find the answers we need, we'll be happy to reopen the issue.
